qa/L0_config_search/test.sh

```diff
@@ -122,6 +142,18 @@ for launch_mode in $TRITON_LAUNCH_MODES; do
             RET=1
         fi
 
+        # Check that GPUs don't match and we return an error
+        if [ $launch_mode != 'remote' ]; then
+            sed -i 's/GPU-/GPU-1-/g' $CHECKPOINT_DIRECTORY/1.ckpt
+            
+            run_analyzer
+            if [ $? -e 0 ]; then
```

Comment:
Also re-do the grep above and make sure it does NOT match

Reply:
Done